pkg/cli/setup/application.go

```diff
@@ -47,6 +47,18 @@ resource demo 'Applications.Core/containers@2023-10-01-preview' = {
 	radYamlTemplate = `workspace:
   application: %q
 ` // Trailing newline intentional.
+
+	bicepConfigTemplate = `{
+	"experimentalFeaturesEnabled": {
+		"extensibility": true,
+		"providerRegistry": true,
+		"dynamicTypeLoading": true
+	},
+	"providers": {
+		"radius": "br:biceptypes.azurecr.io/radius:latest",
+		"aws": "br:biceptypes.azurecr.io/aws:latest"
```

Comment:
Will we end up publishing these to the official ACR registry? Or will we have our own?

Reply:
The plan was to have them pushed to an ACR within OSS assets. Do you think it'd be better to have them in the official registry?

Reply:
I don't have a strong opinion, I just wanted to know the plan 👍

Reply:
This looks good